config/default.yml

```diff
@@ -1972,7 +1973,8 @@ Naming/HeredocDelimiterNaming:
   StyleGuide: '#heredoc-delimiters'
   Enabled: true
   VersionAdded: '0.50'
-  Blacklist:
+  VersionChanged: '0.77'
+  Denylist:
```

Comment:
This should probably be `ForbiddenDelimiters/Patterns`. I'm not sure if the config supports both strings and regular expressions.

Reply:
Sure. Both are acceptable because they are replaced internally by regular expressions :-) I will change to `ForbiddenDelimiters`.
https://github.com/rubocop-hq/rubocop/blob/v0.76.0/lib/rubocop/cop/naming/heredoc_delimiter_naming.rb#L45